### ISSUE TITLE
Restructuring of QWIPU directories

### ISSUE THREAD

**Issue by srt1** [labordynamicsinstitute/qwi_schemas#83]
Heath has proposed that the QWIPU files be reorganized to eliminate the DVD directories, and change the nesting ordering to be more parallel to J2J and LODES. This would mean reordering the nesting on the national QWI, and potentially also mean moving the state qwi files into the /data directory of the website, rather than /pub. The primary reason to do this is to eliminate the second copy of the state qwipu files (though possibly this could be done through symbolic links, if permitted?). 

This could also have implications for the version.txt and manifest files, which are currently separate by DVD directory. It makes little sense to have separated files, if the data files are all sitting in the same directory. Plus, there was never much reason to have separate version files for the demographic characteristics, since there is no design to ever have those different. The F/FA/FS is the only dimension on which the date ranges differ.

If we go ahead with the current Redmine ticket (number 1746), we may have some things to adjust in the schema, as far as the location and organization of the csv files.

**Comment by srt1** [comment]
From: Camille S Norwood (CENSUS/CES FED) <Camille.S.Norwood@census.gov>
Sent: Thursday, February 15, 2018 4:06:10 PM
To: Patrick Hayward (CENSUS/CES FED)
Cc: Chaoling Zheng (CENSUS/CES FED); Lars Vilhuber; Matthew Graham (CENSUS/CES FED); Stephen R Tibbets (CENSUS/CES FED)
Subject: Re: Proposal to restructure QWIPU directories
 

Proof-of-Concept Vintage Ready for Review:  /data/production/dev5/run/qwipu/de/20180215_1003


    This vintage has all DVD subdirectories removed


    This vintage has stopped generating the ISO file

    This vintage has collapsed all:

        version files into one version.txt file

        manifest files into one qwipu_<state>_manifest.txt file

        md5sum files into one qwi_<state>.txt file

________________________________________________

**Comment by larsvilhuber** [comment]
Also see discussion about CONTENT of version.txt (now version_qwi.txt) in #91

**Comment by larsvilhuber** [comment]
Of course, now that I look at it, we haven't actually described the download locations in the past...

I will add something to the CSV_Naming file, which SHOULD actually describe the full path.

**Comment by larsvilhuber** [comment]
Links:
- https://lehd.ces.census.gov/announcements.html#030618
- https://lehd.ces.census.gov/announcements.html#051418
